app/models/user.rb

```diff
@@ -1,12 +1,19 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] `:confirmable` shouldn't be commented on in the user model, this should work together with the 
``` 
     t.string   :confirmation_token
      t.datetime :confirmed_at
      t.datetime :confirmation_sent_at
      t.string   :unconfirmed_email 
```

Reply:
- Well resolved 👍🏽